examples/github-api/README.md

```diff
@@ -0,0 +1,29 @@
+This is an example of using the uplink client. This is an API written in flask that gets the top <br>
```

Comment:
Any particular reason why you're explicitly `<br>` breaking here? 

Given this will be hosted on GitHub, I think we can let the site's Markdown viewer handle soft wrapping paragraphs elegantly. Thoughts?

Reply:
Nope! I am not that great at markdown ;). will change!